controlpanel/frontend/consumers.py

```diff
@@ -38,6 +38,11 @@ def __init__(self, *args, **kwargs):
         """
         self.streaming = True
 
+        user = self.scope.get('user')
```

Comment:
if the events are filtered by the users id then is this required?

Reply:
It is, because `AnonymousUser` doesn't have a `auth0_id` field

Reply:
Of course, there are several ways to handle this - I could have changed `user.auth0_id` to `getattr(user, 'auth0_id', None)` I suppose.